### DISCUSSION TITLE
Port blocking to avoid being bricked

### DISCUSSION THREAD

**Discussion by gbjbasdw** [GerryDazoo/Slinger#369] [General]
I've never done this before so I just wated to check I have got this right.

In my router settings I can block a range of port numbers and enter an IP address so I entered 1-65535 for 192.168.0.21 for my Slingbox. So firstly is this right ?  

I have an issue with getting a static IP for my Slingbox ( see previous post ) so I tried to add another rule - again for 1-65535 but for 192.168.0.254 which is the IP address that my Slingbox is getting allocated ( Dont know why)  The router said I could not do this as the range overlapped an existing range. This seems odd as I thought the blocked range was associated with an IP address. Is it the router being daft or is it me just not understanding this ?

**Comment by bradut** [comment]
@gbjbasdw I have the same issue with my SlingBox's static IP .
After it has been bricked, it no longer got the initially assigned static IP, but 192.168.x.254.
So, I re-wrote the rules in my router to match this static IP and blocked only the ports 5678, 5680, 3724, 3490, 3074 _for this IP only_.

**Reply by proudx2** [reply]
When you perform a reset the slingbox reverts from static ip to dhcp. The prohd guide list all outbound destination ports the prohd uses.  Bricking appears to come from port 5678. Blocking just  port 5678 outbound from slingbox will stop bricking.

**Comment by gbjbasdw** [comment]
That's really useful to know - I thought it was me going nuts.

Seems like another good way to determine if a Slingbox has been bricked.

I have to wonder if Slingmedia bricking boxes is illegal. They are after all damaging property that does not belong to them. I'd call it criminal damage/cyber terrorism. Maybe we need a class action against them ?

Cheers,

Steve

**Comment by gbjbasdw** [comment]
I'm a bit confused with port blocking.
I can add a rule for port 5678 to 5678 protocol TCP&UDP for 192.168.0.21 ok but it wont let me add a second rule for the same port but with a different IP address like 192.168.0.254 
Is this how it normally works or is it  quirk of my Virginmedia router ?

**Reply by GerryDazoo** [reply]
You only need to block 5678 outgoing for the slingbox IP.

**Reply by Toei79** [reply]
> You only need to block 5678 outgoing for the slingbox IP.

where its this done in port forwarding or firewall?

![12-04-2022 (162735)](https://user-images.githubusercontent.com/79808610/205516610-7b46c053-ef9e-4746-bf70-914ee1752489.jpg)

**Reply by regmar505** [reply]
It looks like you have an Asus  router, so select Firewall and Network Service Filter tab and replace 10.0.0.40 by your Slingbox IP.

![image](https://user-images.githubusercontent.com/48108822/205517038-e98f33a5-dad6-4487-97fa-a2b2708a5062.png)

**Reply by Toei79** [reply]
> It looks like you have an Asus router, so select Firewall and Network Service Filter tab and replace 10.0.0.40 by your Slingbox IP.
> 
> ![image](https://user-images.githubusercontent.com/48108822/205517038-e98f33a5-dad6-4487-97fa-a2b2708a5062.png)

thanks i was looking for that i just copy yours

its enough blocking only this port? i see many discussions talking other ports

where i can see how try to access? curious to see if mr sling or someone else.

**Reply by sp222222** [reply]
"where can I see" regmar gifted you the settings in that screenshot ! Not sure how virginMedia router handles rule setup.  Gerry himself answered above stating so; 5678 is only port needed to block mr.sling from bricking your slingbox.  we don't want you to end up with a doorstop :) 
read these steps here - thread readers
https://github.com/GerryDazoo/Slinger/discussions/413

**Reply by Toei79** [reply]
Thanks , I wish can be able use this on tvheadend. There lots of apps for tvh  that let you use remotely. 

I able see all my ota channels through  this plus with asus router you don't need open ports

Also someone suggest a remote can you use remotely  so helps in that part for switch channels

**Reply by bradut** [reply]
I tried the above settings to block ports on an ASUS router, but got bricked in a few minutes.
Noticed this active connection between SlingBox and Sling server, despite I also blocked port 3490:
```
Proto NATed Address                            Destination Address                      State 
tcp   192.168.50.10:7752                       67.148.153.136:3490                      ESTABLISHED
```
![Router Asus 08 - 08a - Blocare porturi - nu functioneaza = Jurnal de sistem - Conexiuni](https://user-images.githubusercontent.com/2496596/209024626-344a452f-b915-4f73-be5d-80e6bd11a652.jpg)

Not sure if destination IP should be `*.*.*.*`  as in the example above.
I left it blank when got bricked, and now I added with asterisks but will wait until it's day time on the other side of the world to see if worked.

**Comment by JLBla** [comment]
In the example above for the Asus router, when replacing 10.0.0.40 by your Slingbox IP, is the Slingbox IP the IP address of the device that you are running the Sling server on, so my iPad, my laptop, my iphone, etc.? Or is the Slingbox IP the IP address of my actual slingbox device.  I just wanted to be sure that I was supposed to block each of my possible devices that I may use to run the Slinger from port 5678.

**Reply by regmar505** [reply]
Slingbox IP of your actual slingbox device on your LAN

**Reply by JLBla** [reply]
Thank you.  For the destination IP, do I type in the symbol for 4 asterisks separated by periods, or does that mean leave it blank?  Sorry, I just wanted to extra careful that I am blocking port 5678 correctly.

**Reply by regmar505** [reply]
Yes you type 4 asterisks separated by periods which means all possible addresses

**Reply by sp222222** [reply]
i had two asus routers barely different model - one took 0.0.0.0/0 the other one took *.*.*.* - go figure

**Comment by womeara6** [comment]
Hi Bradut. What model slingbox got bricked? Thanks, BillSent from my iPhoneOn Dec 21, 2022, at 7:03 PM, Bradut Dima ***@***.***> wrote:﻿
I tried the above settings to block ports on an ASUS router, but got bricked in a few minutes.
Noticed this active connection between SlingBox and Sling server, despite I also blocked port 3490:
Proto NATed Address                            Destination Address                      State 
tcp   192.168.50.10:7752                       67.148.153.136:3490                      ESTABLISHED


Not sure if destination IP should be *.*.*.*  as in the example above.
I left it blank when got bricked, and now I added with asterisks but will wait until it's day time on the other side of the world to see if worked.

**Reply by bradut** [reply]
Hi @womeara6 
Sorry for my late reply, I just returned home.
The Slingbox is a PRO-HD which I tried to set up at my remote location where they have an ASUS router.

**Reply by womeara6** [reply]
Thanks.  I have a 500 so I was interested if that model was now on the endangered species list.
